IntegrationTests/IntegrationScenarios/Scenarios/SessionReplay/SRMultipleViewsRecordingScenarioTests.swift

```diff
@@ -13,6 +13,11 @@ private extension ExampleApplication {
         buttons["NEXT"].safeTap(within: 5)
     }
 
+    /// Triggers SR recording
+    func deadClick() {
+        windows.firstMatch.tap()
+    }
```

Comment:
Why do we need this now?

Reply:
Still needed as integration tests view are quite static.